core/flow/rule_manager.go

```diff
@@ -3,42 +3,154 @@ package flow
 import (
 	"encoding/json"
 	"fmt"
+	"sync"
+
 	"github.com/alibaba/sentinel-golang/logging"
 	"github.com/pkg/errors"
-	"sync"
 )
 
-// const
 var (
 	logger = logging.GetDefaultLogger()
 )
 
+// SafeTrafficControllers represents the threadSafe map storage for TrafficShapingController.
+type SafeTrafficControllers struct {
```

Comment:
Is it better to rename to ConcurrentTrafficControllers?

Reply:
That sounds better.